app/Auth.tsx

```diff
@@ -23,22 +29,28 @@ import { CustomModal } from '../components/utils/CustomModal';
 import Registration from '../components/auth/Registration';
 
 export default function Auth() {
+
   const colorScheme = useColorScheme() ?? 'light';
   const dispatch = useAppDispatch();
   const router = useRouter();
   const [regUI, setRegUI] = useState<boolean>(true);
   const phoneNumber = useAppSelector(selectPhoneNumber);
   const firstName = useAppSelector(selectFirstName);
   const lastName = useAppSelector(selectLastName);
+  const selectedCompostStand = useAppSelector(selectCompostStand);
+  const isRegButtonDisabled = !phoneNumber || (regUI && (!firstName || !lastName)) || !selectedCompostStand;
```

Comment:
Maybe:
```suggestion
  const isRegButtonDisabled = !phoneNumber || (regUI && (!firstName || !lastName || !selectedCompostStand));
```
We only require the user to select a stand if they are registering. Is it possible the current format of the conditional will require users who are logging in to add a compost stand?

Reply:
You're right :)